modules/aws/ec2.go

```diff
@@ -221,6 +221,68 @@ func TerminateInstanceE(t *testing.T, region string, instanceID string) error {
 	return err
 }
 
+// GetAmiPubliclyAccessible returns whether the AMI is publicly accessible or not
+func GetAmiPubliclyAccessible(t *testing.T, awsRegion string, amiID string) bool {
+	output, err := GetAmiPubliclyAccessibleE(t, awsRegion, amiID)
+	if err != nil {
+		t.Fatal(err)
+	}
+	return output
+}
+
+// GetAmiPubliclyAccessibleE returns whether the AMI is publicly accessible or not
+func GetAmiPubliclyAccessibleE(t *testing.T, awsRegion string, amiID string) (bool, error) {
+	launchPermissions, err := GetLaunchPermissions(t, awsRegion, amiID)
+	if err != nil {
+		return false, err
+	}
+	for _, launchPermission := range launchPermissions {
+		if aws.StringValue(launchPermission.Group) == "all" {
+			return true, nil
+		}
+	}
+	return false, nil
+}
+
+// GetAccountsWithLaunchPermissions returns list of accounts that the AMI is shared with
+func GetAccountsWithLaunchPermissions(t *testing.T, awsRegion string, amiID string) []string {
+	output, err := GetAccountsWithLaunchPermissionsE(t, awsRegion, amiID)
+	if err != nil {
+		t.Fatal(err)
+	}
+	return output
+}
+
+// GetAccountsWithLaunchPermissionsE returns list of accounts that the AMI is shared with
+func GetAccountsWithLaunchPermissionsE(t *testing.T, awsRegion string, amiID string) ([]string, error) {
+	accountIDs := []string{}
+	launchPermissions, err := GetLaunchPermissions(t, awsRegion, amiID)
+	if err != nil {
+		return accountIDs, err
+	}
+	for _, launchPermission := range launchPermissions {
+		if aws.StringValue(launchPermission.UserId) != "" {
+			accountIDs = append(accountIDs, aws.StringValue(launchPermission.UserId))
+		}
+	}
+	return accountIDs, nil
+}
+
+// GetLaunchPermissions returns launchPermissions as configured in AWS
+func GetLaunchPermissions(t *testing.T, awsRegion string, amiID string) ([]*ec2.LaunchPermission, error) {
```

Comment:
By convention, should this method be called `GetLaunchPermissionsE` since it returns an error?

Reply:
fixed